lib/service/Service.ts

```diff
@@ -0,0 +1,86 @@
+import assert from 'assert';
+import { GrpcComponents } from '../grpc/GrpcServer';
+import Logger from '../Logger';
+import Pool from '../p2p/Pool';
+import OrderBook from '../orderbook/OrderBook';
+import { orders, matchingEngine } from '../types';
+import LndClient from '../lndclient/LndClient';
+import RaidenClient, { TokenSwapPayload } from '../raidenclient/RaidenClient';
+import grpc from 'grpc';
+import { PairInstance } from '../types/db';
+import { GetInfoResponse } from '../proto/lndrpc_pb';
+import { Orders } from '../orderbook/OrderBookRepository';
+import { OwnOrder } from '../types/orders';
+
+export type XudInfo = {
+  lnd: GetInfoResponse,
+};
+
+/** Class containing the available RPC methods for XUD */
+class GrpcService {
```

Comment:
this class should have nothing to do with gRPC.. so perhaps `Service` instead of `GrpcService`

Reply:
Ah yes this was just an oversight, good catch.